tests/fixtures/netexidf2ntfs/input/netex/arrets.xml

```diff
@@ -99,6 +99,33 @@
                                 <QuayRef ref="FR:91272:ZDE:50076651:STIF" version="732658"/>
                             </quays>
                         </StopPlace>
+                        <StopPlace dataSourceRef="STIF-REFLEX:Operator:59" version="902482" created="2015-02-20T04:02:10.0Z" changed="2017-12-19T10:12:38.0Z" id="will_be_removed_by_sanitize">
+                            <keyList>
+                                <KeyValue typeOfKey="OBJET_QUALIFIER">
```

Comment:
```suggestion
                                <KeyValue typeOfKey="OBJECT_QUALIFIER">
```
But in reality, it doesn't matter what this key is, right? So maybe we can just remove it entirely (and all the other instances above and below) so the fixture can be minimalist.
```suggestion
                                <KeyValue>
```

Reply:
I just copied data from a real Netex file. I'm not sure if we have to remove xml nodes not used.
I prefer to have a minimal fixture but with a complete structure for integration tests
What do you think @ArnaudOggy ?

Reply:
@datanel 
Good question. I have no opinion really decided but I think like you; a minimal but complete fixture.